ex8_ch_3.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+people = int(input("How many people will eat pizza? "))
+number_of_pizzas = int(input("How many pizzas do you have? "))
+slices_per_pizza = int(input("How many slices are in each pizza? "))
+
+total_slices = slices_per_pizza * number_of_pizzas
+
+print(f"There are {people} people and {number_of_pizzas} pizzas.")
+
+pizza_slices_per_person = total_slices / people
+print(f"Each person gets {pizza_slices_per_person} pieces of pizza.")
+
+leftover_pieces = total_slices % people
+print(f"There are {int(leftover_pieces)} leftover pieces.")
+
```

Comment:
How do I make the output pluralization correct? Example: Each person gets 2 pieces of pizza. or
Each person gets 1 piece of pizza.

Reply:
This is the `if-else` that you're almost at! But here's a sneak peek:

```
if leftover_pieces == 1:
  print(f"Each person gets 1 piece of pizza.")
else:
   print(f"Each person gets {leftover_pieces) pieces of pizza.")
```